technologies/job/r/r-3.4.4-base/entrypoint

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
 
+# Change default CRAN to ENV VAR if present
+if [[ -z "${R_CUSTOM_CRAN}" ]]; then
+    echo "No custom CRAN defined, using default, you can configure one using R_CUSTOM_CRAN env var"
+else
+    sed -i 's#https://cloud.r-project.org#'"${R_CUSTOM_CRAN}"'#g' /usr/lib/R/etc/Rprofile.site
+fi
```

Comment:
I suggest to rename the variable `R_CRAN_URL`.
In a near future this variable will be `https://cloud.r-project.org` by default :-)
```suggestion
if [[ -z "${R_CRAN_URL}" ]]; then
    echo "No custom CRAN defined, using default, you can configure one using R_CRAN_URL env var"
else
    sed -i 's#https://cloud.r-project.org#'"${R_CRAN_URL}"'#g' /usr/lib/R/etc/Rprofile.site
fi
```